[*] Running ...
[*] Dataset loaded: D:\ProgramLanguages\Programs\Python\code-gen\reevo/problems/tsp_gls\dataset/train200_dataset.npy with 10 instances.
D:\ProgramLanguages\Programs\Python\code-gen\reevo\problems\tsp_gls\gpt.py:18: RuntimeWarning: divide by zero encountered in scalar divide
  
D:\ProgramLanguages\Programs\Python\code-gen\reevo\problems\tsp_gls\gpt.py:29: RuntimeWarning: invalid value encountered in scalar multiply
  if max_feature > 0:
[*] Instance 0: 10.560360211540674
[*] Instance 1: 10.70721544023672
[*] Instance 2: 10.87485845140353
[*] Instance 3: 10.425726994419115
[*] Instance 4: 10.614171446582915
[*] Instance 5: 10.411041780676639
[*] Instance 6: 11.105239282462154
[*] Instance 7: 10.691543950027732
[*] Instance 8: 10.593058912264457
[*] Instance 9: 10.269406168775552
[*] Average:
10.625262263838948
